public/js/monitor.js

```diff
@@ -264,38 +295,46 @@ function doButtonRouting(event) {
 
 //re-enables inputs and clears loader
 function restoreInputs() {
+  for (const form of document.forms) {
+    form.classList.remove("loading-data");
+    form.classList.remove("invalid");
+  }
   for (const input of document.querySelectorAll("input")) {
     if (input.disabled) {
       input.disabled = false;
     }
   }
-  removeClass("form", "loading-data");
-  removeClass("form", "invalid");
 }
 
-
 //prevents footer from covering stuff up
-function bodyPadding() {
-  const footerHeight = document.getElementById("footer").offsetHeight;
-  document.body.style.paddingBottom = footerHeight + 60 + "px";
-}
 
 window.addEventListener("pageshow", function() {
-  bodyPadding();
   if (document.getElementById("no-breaches") || document.getElementById("found-breaches")) {
-    sendGAPing("Pageview");
+    ga_sendPing("Pageview");
   }
   if (document.forms) {
     restoreInputs();
   }
 });
 
-window.addEventListener("resize", bodyPadding);
-
 if(document.forms) {
   addFormListeners();
 }
 
+if(document.getElementById("confirmation")) {
+  document.querySelector("header").querySelector(".social-media-sharing-buttons").classList.add("hide");
+}
+
+if (document.getElementById("no-breaches")) {
+  document.getElementById("scan-another-email").classList.add("banner");
+}
+
+if(document.querySelectorAll(".sendGA")) {
```

Comment:
Do we need this `if`? Or will the `for (const el of document.querySelectorAll(".sendGA")) {..}` just silenty skip if there are no found ".sendGA" elements? Or can/should we cache the result of `document.querySelectorAll(".sendGA")` instead of traversing the DOM twice?

Reply:
removed with no issues.